typing/shape_reduce.mli

```diff
@@ -0,0 +1,60 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                Ulysse Gérard, Thomas Refis, Tarides                    *)
```

Comment:
I am wondering now that the `reduce` part is its own module if this copyright header should mention @gasche and @Ekdohibs ?

Reply:
Sounds right. @gasche, @Ekdohibs, how should I attribute you ?

Reply:
The affiliation at the time of doing the bulk of the work on these files would be:

```
(* Gabriel Scherer, projet Partout, INRIA Saclay *)
(* Nathanaëlle Courant, OProjet Cambium, INRIA Paris *)
```

the affiliations today would be

```
(* Gabriel Scherer, projet Picube, INRIA Paris *)
(* Nathanaëlle Courant, OCamlPro *)
```